go.mod

```diff
@@ -43,7 +43,7 @@ require (
 	code.cloudfoundry.org/bbs v0.0.0-20200403215808-d7bc971db0db
 	code.cloudfoundry.org/garden v0.0.0-20210208153517-580cadd489d2
 	code.cloudfoundry.org/lager v2.0.0+incompatible
-	github.com/DataDog/agent-payload/v5 v5.0.36
+	github.com/DataDog/agent-payload/v5 v5.0.38-0.20221027070638-e53bb02f8e9e
```

Comment:
Can you link to the agent payload change?

Reply:
https://github.com/DataDog/agent-payload/pull/193